### ISSUE TITLE
Logging validation loss to tensorboard

### ISSUE THREAD

**Issue by rafaelvalle** [jik876/hifi-gan#9]
Currently the code does not log validation loss to tensorboard. 
This way we do not know in what optimization regime we are, e. g. underfitting or overfitting.
Can you please add this feature?

**Comment by jik876** [comment]
I've added the mel-spectrogram error used as loss in training to logging. 
While this is not highly correlated with perceptual quality, it would be helpful to see the optimization tendency in training.
